### ISSUE TITLE
Config Hooks to allow "go back" to a specific page

### ISSUE THREAD

**Issue by folletto** [WordPress/gutenberg#21795]
Gutenberg provides a somehow isolated experience from WP Admin, and it would be nice if we allowed developers to make use of Gutenberg without the risk of "breaking the user flow" and redirecting them back to WP Admin.

![Screenshot 2020-04-22 at 19 00 36](https://user-images.githubusercontent.com/4389/80016871-a0d89f00-84cb-11ea-84eb-d1253a632403.png)

It would be nice to have a configuration hook that when activated did two things:

1. Locked Gutenberg to be full screen only
2. Changed the page where the (W) goes back to

This change would allow a variety of uses:

* Agencies could have their own workflow and provide a different listing page
* Products and plugins using Gutenberg could use Gutenberg and make sure after the user would go back to them

In short, it would make Gutenberg to be already more easily integrated.

**Comment by Soean** [comment]
Have you seen #21121, which was merged today?

**Comment by folletto** [comment]
Yep, nice improvement! :) 

Maybe I'm missing something, I don't get how it is related?

**Comment by talldan** [comment]
Seems related to https://github.com/WordPress/gutenberg/issues/20929 in allowing customisation of this part of the UI.

**Comment by folletto** [comment]
Yes, it's probably a duplicate of that one.
I'm closing this one and adding details on the other.
